atlanticForestCode.R

```diff
@@ -0,0 +1,179 @@
+## Preliminaries
+
+rm(list=ls())
+
+carbonToCO2 <- 44/12 # (44 units CO2/12 units C) from https://www.epa.gov/energy/greenhouse-gases-equivalencies-calculator-calculations-and-references
+tonsCO2ToDollars2040 <- 103 #from https://www.whitehouse.gov/wp-content/uploads/2021/02/TechnicalSupportDocument_SocialCostofCarbonMethaneNitrousOxide.pdf?source=email Using 2.5% discount rate for 2040 emissions
+tonsCO2ToDollars2020 <- 76 #from https://www.whitehouse.gov/wp-content/uploads/2021/02/TechnicalSupportDocument_SocialCostofCarbonMethaneNitrousOxide.pdf?source=email Using 2.5% discount rate for 2040 emissions
+carbonToDollars2020 <- carbonToCO2 * tonsCO2ToDollars2020
+carbonToDollars2040 <- carbonToCO2 * tonsCO2ToDollars2040  
+hectaresAtlanticForest <- 142274200 #From https://mapbiomas.org/en/mapbiomas-launches-1
+baseYear <- 2010
+
+library(ggplot2)
+library(data.table)
+library(lfe)
```

Comment:
This lfe library is not actually used in the script, we think we can safely remove it